packages/xrpl/package.json

```diff
@@ -27,7 +27,8 @@
     "ripple-address-codec": "^4.1.1",
     "ripple-binary-codec": "^1.1.3",
     "ripple-keypairs": "^1.0.3",
-    "ws": "^8.2.2"
+    "ws": "^8.2.2",
+    "xrpl-secret-numbers": "^0.3.3"
```

Comment:
should this be included in the monorepo?

Reply:
Maybe wait to bring it up to the top level until another package needs it?

Reply:
no i mean, should `xrpl-secret-numbers`'s source be in this monorepo. not sure where else it's used or how big it is, but that might make sense

Reply:
I am sure the XRPL-Labs team uses this lib in their XUMM app. I think @WietseWind is the maintainer?
For now I would leave it as "external" import and would not add it to the monorepo. but maybe Wietse thinks this would be a good idea :)

Reply:
@ledhed2222 It's a fairly small package, and the (non dev) dependencies overlap with `ripple-keypairs` (brorand, ripple-keypairs)

Reply:
that would be a good reason to maintain it within the monorepo :)

IMO, the criteria for whether it should be part of the monorepo:

1) is the purpose/scope of the package overlapping with the monorepo's? in this case i think it might be
2) does it depend on other packages w/in this monorepo, potentially requiring work on it and `ripple-keypairs` simultaneously.